Userland/Libraries/LibVideo/VP9/ContextStorage.h

```diff
@@ -194,8 +194,7 @@ class Vector2D {
 private:
     void clear_storage()
     {
-        if (m_storage)
-            free(m_storage);
+        delete[] m_storage;
```

Comment:
shouldnt this still be behind the pointer check?
(also, lets be nice and make it a sized delete, if you can figure out the syntax, the docs are surprisingly unhelpful with that)

Reply:
No, you can delete a null pointer, and I removed the check because clangd said as much:
![clangd](https://user-images.githubusercontent.com/5600524/234069431-854205f9-07c0-449e-b0cc-f662d39145b4.png)

Reply:
The more you know